.github/workflows/tests.yml

```
@@ -0,0 +1,33 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: tests | coverage
+
+on:
+  push:
+    branches: [ "main", "dev" ]
+  pull_request:
+    branches: [ "main", "dev" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .[dev]
+    - name: Unit and coverage tests
+      run: |
+        pytest tests/menelaus --cov=src/ --cov-report term
+        coverage report -m --fail-under=99
```

```suggestion
        coverage report -m --fail-under=100
```

Once we get #45 fixed, we should be at 100% coverage. We can force-merge in this instance, and then not need to make a different PR to fix this constant later.